salt/utils/openstack/nova.py

```diff
@@ -259,9 +270,11 @@ def __init__(
                            os_auth_plugin=os_auth_plugin,
                            **kwargs)
 
-    def _new_init(self, username, project_id, auth_url, region_name, password, os_auth_plugin, auth=None, verify=True, **kwargs):
+    def _new_init(self, username, project_id, auth_url, region_name, password, os_auth_plugin, auth=None, verify=True,
+                  **kwargs):
         if auth is None:
             auth = {}
+        verify = kwargs.get('verify', False)
```

Comment:
This doesn't make sense, how would verify get in `kwargs` when it's defined as a keyword argument on line 273?